extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -73,6 +75,9 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				slotTargeting.rv = count.toString();
 			}
 		}
+		if (slot.name === 'BOTTOM_LEADERBOARD' && win.ads.runtime.uap) {
+			slotTargeting.uap = win.ads.runtime.uap.toString();
```

Comment:
Can we set this value on page level (when BFAA template is called)? Then we could get rid of extra `ads.runtime.uap` and it would be beneficial for floating medrec.